### PR TITLE
Match func_80142440 (z_vr_box.c OK)

### PR DESCRIPTION
**Pull request by engineer124**
I distinctly remember this function from OoT being a pain. Thankfully, it was near copy-paste to MM.

### CODE REVIEW

**Review by hensldm — APPROVED** [review]
LGTM :)

**Comment by engineer124** [comment]
> LGTM :)

Just wanted to note that I just added some small docs and cleaned up some matchings for some skybox functions

**Review by AngheloAlf — APPROVED** [review]

File: src/code/z_vr_box.c [hunk 1]
```diff
@@ -180,114 +183,120 @@ void func_80143148(SkyboxContext* skyboxCtx, s32 arg1) {
     }
 }
 
-void func_801431E8(GameState* gameState, SkyboxContext* skyboxCtx, s16 skyType) {
+void Skybox_Setup(GameState* gameState, SkyboxContext* skyboxCtx, s16 skyboxId) {
     PlayState* play = (PlayState*)gameState;
     size_t size;
     void* offset;
```
Comment (hensldm) [inline src/code/z_vr_box.c:189]: Kind of small, but this doesn't seem to be an offset. Perhaps `segment`?

**Review by hensldm — APPROVED** [review]
1 thought

File: include/z64.h [hunk 2]
```diff
@@ -645,9 +645,9 @@ typedef struct {
 
 typedef struct {
     /* 0x000 */ View view;
-    /* 0x168 */ void* skyboxStaticSegment[4];
+    /* 0x168 */ void* staticSegments[4];
     /* 0x178 */ void* skyboxPaletteStaticSegment;
```
Comment (EllipticEllipsis) [inline include/z64.h:649]: Might as well drop the `skybox` from this one too.

**Review by EllipticEllipsis — APPROVED** [review]

**Pull request merged**